Processes/Docs-Contribution-Process-External.md

```diff
@@ -0,0 +1,44 @@
+# Docs Contribution Process - External
```

Comment:
This entire document should be merged with the "internal" document. We don't want to separate documents to maintain with similar info. The only thing that changes between the two should be how they create branches and pull requests

Reply:
I will use the Internal document, change the title, and adapt it to both uses.

Reply:
Done